lib/sdam/topology.js

```diff
@@ -223,7 +251,8 @@ class Topology extends EventEmitter {
    * Initiate server connect
    *
    * @param {object} [options] Optional settings
-   * @param {Array} [options.auth=null] Array of auth options to apply on connect
+   * @param {Array} [options.auth] Array of auth options to apply on connect
+   * @param {any} [options.readPreference] TODO:(ts) types and documentation
```

Comment:
I'll stop comment on these now, but remember this goes right into _user-facing_ documentation. Imagine seeing that

Reply:
Indeed, I'll resolve these